source/run-command.txt

```diff
@@ -0,0 +1,248 @@
+            }, 
+            'me': 'cluster0-shard-00-02.fxoii.mongodb.net:27017', 
+            'electionId': ObjectId('7fffffff00000000000000e3'), 
+            'lastWrite': {
+                'opTime': {
+                    'ts': Timestamp(1730486145, 22), 
+                    't': 227
+                }, 
+                'lastWriteDate': datetime.datetime(2024, 11, 1, 18, 35, 45), 
+                'majorityOpTime': {
+                    'ts': Timestamp(1730486145, 22), 
+                    't': 227
+                }, 
+                'majorityWriteDate': datetime.datetime(2024, 11, 1, 18, 35, 45)
+            }, 
+            'maxBsonObjectSize': 16777216, 
+            'maxMessageSizeBytes': 48000000, 
+            'maxWriteBatchSize': 100000, 
+            'localTime': datetime.datetime(2024, 11, 1, 18, 35, 45, 309000), 
+            'logicalSessionTimeoutMinutes': 30, 
+            'connectionId': 23889, 
+            'minWireVersion': 0,
+            'maxWireVersion': 21, 
+            'readOnly': False, 
+            'ok': 1.0, 
+            '$clusterTime': {
+                'clusterTime': Timestamp(1730486145, 22), 
+                'signature': {
+                    'hash': b"\x1a\xf7{>q%F\xc2\x89\x15\x13W29\x91\xaae'~\xe4", 
+                    'keyId': 7379292132843978793
+                }
+            }, 
+            'operationTime': Timestamp(1730486145, 22)
+        }
+
+To find a link to a full list of database commands and corresponding
+parameters, see the :ref:`Additional Information section
+<pymongo-addtl-info-runcommand>`.
+
+.. _pymongo-command-response:
+
+Command Cursor
+--------------
+
+The ``command()`` method returns the result of the command that was run.
+You can also use the ``cursor_command()`` method, which issues a MongoDB 
+command and parses the response as a `CommandCursor <{+api-root+}pymongo/command_cursor.html#pymongo.command_cursor.CommandCursor>`__.
+The ``CommandCursor`` can be used to iterate over command results.
+
+The following example uses the ``cursor_command()`` method on the ``sample_mflix``
+database. It runs the ``find`` command on the ``movies`` database to filter by
+documents in which the ``runtime``field has a value of ``11``.
+
+.. io-code-block::
+    :copyable: true
+
+    .. input:: /includes/run-command.py
+        :language: python
+        :dedent:
+        :start-after: start-cursor-command
+        :end-before: end-cursor-command
+
+    .. output::
+        :language: json
+        :visible: false
+
+        { 
+            '_id': ObjectId('573a1390f29313caabcd42e8'),
+            'runtime': 11, 
+            'title': 'The Great Train Robbery',
+            ...
+        },
+        {
+            {'_id': ObjectId('573a1394f29313caabce0f10'),
+            'runtime': 11, 
+            'title': 'Glas',
+            ...
+        },
+        ...
+
+Before you run a command, learn about the response format of the command so that
```

Comment:
This direction feels a little incomplete without linking to a resource on this or including more info about responses.